app/src/main/res/layout/fragment_details.xml

```diff
@@ -92,10 +91,14 @@
                 android:layout_below="@id/col2"
                 android:layout_marginStart="20dp"
                 android:layout_marginTop="40dp"
-                android:layout_marginEnd="20dp"
-                android:clickable="false"
-                android:filterTouchesWhenObscured="true" />
+                android:layout_marginEnd="20dp"/>
 
+        <View
+                android:layout_width="wrap_content"
+                android:layout_height="100dp"
```

Comment:
Don't hardcode the height. Since the parent layout is a relative layout you can write the code in the layout such that only dayPicker is covered.